_posts/2016-09-22-professional-development-at-artsy-engineering.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Professional Development at Artsy Engineering"
+date: 2016-09-22 16:37
+comments: true
+author: alan
+categories: [engineering, professional, development, organization, education]
+---
+
+In considering an offer to join us at Artsy, one of our newest incoming engineers asked me a great question: _How does the tech team do professional development?_
+
+As I thought about it, I began to realize that the answer is “a lot”! Most of our efforts evolved organically. Someone had an idea, and people rallied around it. I thought it would be useful to share, in case others are inspired by what's caught on here. Here are some of the things we do.
+
+<!-- more -->
+
+# Practices
+
+Our engineering team used to be organized along _practice_ lines: web, mobile native, and platform. Following our recent [product reorganization][1], we’re primarily organized in product teams, which are more deeply integrated and aligned with our business units. But practices are here to stay. They serve as horizontal channels for developers working in similar technologies to collaborate and share best practices.
```

Comment:
This could mention that each practice has their own specific "Tech standups", that can focus on discussions specific to that practice - the web ones end up turning into a longer discussion on some tech choices a lot of the time, and the mobile ones tend to end up complaining about apple + government bureaucracy.